### PR TITLE
Create DenkiAo-3244 Markdown Profile

### PR DESCRIPTION
**Pull request by dibarradev**
# Steps to reproduce exercise:

- Fork over [Undefined Academy (semana 1)](https://github.com/undefined-academy/semana-1) Repository.
- Clone in Home folder-
- Create file `DenkiAi-3244.md` into `github-profiles` folder.
- Add profile with direct link to personal GitHub profile.
- Create commit of changes.
- Push commit's into forked repository.
- Create **Pull Request** in original repository.
- Go to the moon 🚀🌝.

### CODE REVIEW

**Review by JoseCamilo84 — CHANGES_REQUESTED** [review]
Hola, creo q te falta crear la carpeta y los 2 archivos html y css

**Comment by dibarradev** [comment]
> Hola, creo q te falta crear la carpeta y los 2 archivos html y css

Hola @JoseCamilo84. Esta práctica es la del día 2, aún no llego a la práctica de HTML 😬

**Review by camilasilvab — APPROVED** [review]

File: DenkiAo-3244/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Hello, dear netizen! 🖖
+
+My name is David (aka *Dave*), I'm currently working as a front-end web developer in this big world. Day by day I strive to learn more and apply it to the projects in which I am present, always looking for that special connection with the end user, making their experience easier and friendlier.
+
+Let's work together! 🔥
+
+**Do you want to visit my GitHub profile? [Click here 🔗](https://github.com/dibarradev/dibarradev)**
```
Comment (Psychobroder) [inline DenkiAo-3244/README.md:7]: Hello Dave, I have seen as a good practice to put the link above the text to which you refer, in this case (github profile)

**Review by Psychobroder — APPROVED** [review]
you have a very clean style to write, keep up the great work!

**Review by gurrutia15 — APPROVED** [review]
Bien hecho, lo único es que la carpeta DenkiAo-3244 debiera ir dentro de github-profiles

File: DenkiAo-3244/README.md (hunk 1 above)
Comment (dibarradev) [reply DenkiAo-3244/README.md:7]: Hi Psychobroder, thanks a lot for the tip, I'll keep that in mind. 👊

**Pull request closed without merging**